packages/expo/src/environment/LogBox.fx.expo.ts

```diff
@@ -1,6 +1,7 @@
 import Constants from 'expo-constants';
 
 if (__DEV__) {
+  // @ts-ignore: TODO: not on the schema
   if (Constants.manifest?.experiments?.redesignedLogBox) {
```

Comment:
should we add `experiments: Record<string, any>` to the schema?

Reply:
Nah we removed it, not sure how much longer we want to keep this warning around for.